contrib/syslog-debun

```diff
@@ -6,6 +7,17 @@
 ### This software may be used and distributed according to the terms of GNU GPLv2
 ### http://www.gnu.org/licenses/gpl-2.0.html
 
+### Check for "local" variable support
+if type local >/dev/null; then
+	:
+elif type bash >/dev/null; then
```

Comment:
Gyuri basically listed the reasons why it's safer to start off with /bin/sh.
I noticed that Solaris 9 only has some ancient bash, but I tested the scripts there as well, and it worked fine there as well. I only used bashishms more freely for the platforms where bash is readily available (Linux systems).